src/entities/Tile.ts

```diff
@@ -8,23 +8,23 @@ export class Tile {
     public readonly id: UUID;
     public readonly type: int;
     public state: Nullable<TileState>;
-    public rotation: Nullable<int>;
+    public rotation: int;
     public coordinateX: Nullable<int>;
     public coordinateY: Nullable<int>;
 
     public constructor(random: seedrandom.PRNG, type: int) {
         this.id = seededUUIDv4(random);
         this.type = type;
         this.state = null;
-        this.rotation = null;
+        this.rotation = 0;
```

Comment:
I have implementet rotation like this: 0 means it isn't rotated, 1 means 90, 2 : 180 3 : 270. When the tile is constructed it is rotation 0, it doesn't have to be null since it can't be anything other then 0 until the player rotates it. It just safes asserts and if you assert something that is 0 it evaluates to false which gave some issues.